frontend/src/components/institute/common/SchoolMove.vue

```diff
@@ -1,22 +1,39 @@
 <template>
   <div id="schoolMove" class="px-0 pt-3 ma-0" style="width: 100%;">
-    <v-row>
-      <v-col class="d-flex justify-end">
-      <PrimaryButton v-if="isMoveSchoolAllowed()" id="moveSchoolButton" class="mr-2" secondary icon-left
-                               icon="mdi-arrow-left-right" @click.native="moveSchool"
-                               text="Move School"></PrimaryButton>
+    <v-row v-if="loading">
```

Comment:
The HTML markup tree seems out of alignment to me, thus making it harder to follow.  It took a minute for me to see which closing row tags matched the opening row tags to ensure everything was opened & closed.

Could we give this file a bit of quick re-format?

Reply:
We got this.